docs/community-operators-troubleshooting.md

```diff
@@ -0,0 +1,219 @@
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="build-bundle"></a>build-bundle
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="make-bundle-repo-public"></a>make-bundle-repo-public
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-supported-versions"></a>get-supported-versions
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="add-bundle-to-index"></a>add-bundle-to-index
+Failures at this step are uncommon and if they do occur they are often transient.  
+
+Please click the `Close pull request` button in GitHub then click the `Reopen pull request` button.
+Closing and re-opening your Pull request will restart the Pipeline. If your PR fails at this step twice in a row please contact Red Hat Support
+
+## <a id="make-index-repo-public"></a>make-index-repo-public
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-ci-results-attempt"></a>get-ci-results-attempt
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="preflight-trigger"></a>preflight-trigger
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+> Note: There is a known issue if you Operator only supports OpenShift 4.7 or below. In this case we recommend using the CI Pipeline.
+
+## <a id="evaluate-preflight-result"></a>evaluate-preflight-result
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-ci-results"></a>get-ci-results
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="link-pull-request-with-open-status"></a>link-pull-request-with-open-status
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="merge-pr"></a>merge-pr
+If hosted pipeline fails at this task with the error message: `Pull request Auto merge is not allowed for this repository (enablePullRequestAutoMerge)` then re-trigger the pipeline by running command `/pipeline restart operator-hosted-pipeline`.  
+
+Another Failure at this step may happen if the pull request is a draft, convert the draft to a pull request and then retry. If this problem persists at this step, contact Red Hat Support.
+
+## <a id="link-pull-request-with-merged-status"></a>link-pull-request-with-merged-status
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+# <a id="released"></a>Operator Released Pipeline
```

Comment:
The release pipeline has many more tasks than what is mentioned below. Please add the missing ones.